src/ttt/core.clj

```diff
@@ -14,18 +15,21 @@
         (fn [current-state]
            (merge current-state {:board board :players players :current-player (first players)}))))
 
-(defn play 
+(defn play
```

Comment:
Let's write tests for this. Now, I know this is probably the most difficult part... Would you like to pair with one of the mentors or someone else from the office on this one to get you started?

Reply:
Agreed - I think this will be a helpful exercise in practicing TDD on things that people often write off as "impossible" (and hence, where the bugs go to hide).  Exposing dependencies, testing with some sort of test double, and driving the algorithm incrementally is often the trick here.  (Sometimes it seems really hard in the beginning, but it winds up not being that hard once you get past the first couple of tests).

Reply:
I think I'm going to try and redo this function test driving it now that I have a better sense of how `recur` works. That was the part that I was stuck on for awhile.